source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1003,6 +1004,19 @@ message TransformationPermuteFunctionParameters {
 
 }
 
+message TransformationPermutePhiOperands {
+
+  // Permutes operands of some OpPhi instruction.
+
+  // Result id of the instruction to apply transformation to.
```

Comment:
"to apply transformation to" -> "to apply the transformation to"

Reply:
Fixed in 0d782ae.